src/main/java/org/breedinginsight/api/v1/controller/ProgramController.java

```diff
@@ -130,6 +130,9 @@ public ProgramController(ProgramService programService, ProgramUserService progr
             Program program = programService.create(programRequest, actingUser);
             Response<Program> response = new Response(program);
             return HttpResponse.ok(response);
+        } catch (AlreadyExistsException e){
```

Comment:
In some of the newer api code I've switched to using HttpStatusException then I don't have to catch anything in the controller and the HttpStatusExceptionHandler will handle it automatically. So for this you could throw a HttpStatusException(HttpStatus.CONFLICT, "Program name already in use") from the program service. Don't have to change this but just wanted to let you know for the future.

Reply:
Good to know. I was following the existing pattern given it already catches UnprocessableEntityException in the controller, but going forward I can use HttpStatusExceptionHandler.